turtle.go

```diff
@@ -84,6 +88,23 @@ type Turtle struct {
 	closed uint32
 }
 
+func (t *Turtle) initBuckets() (err error) {
+	t.mux.Lock()
```

Comment:
Not needed since this is only called from New

Reply:
Fair enough!